README.md

```diff
@@ -10,6 +10,7 @@ Orchard Core consists of two distinct projects:
 [![BSD-3-Clause License](https://img.shields.io/badge/license-BSD--3--Clause-blue.svg)](LICENSE)
 [![Documentation](https://readthedocs.org/projects/orchardcore/badge/)](https://docs.orchardcore.net/)
 [![Crowdin](https://badges.crowdin.net/orchard-core/localized.svg)](https://crowdin.com/project/orchard-core)
+[![Discord](https://img.shields.io/discord/551136772243980291?color=%237289DA&label=OrchardCore&logo=discord&logoColor=white&style=flat-square)](https://orchardcore.net/discord)
```

Comment:
Shouldn't this already work? 
![{4BCD193A-D2C3-4EE0-95B0-B47F8C013B4C}](https://github.com/user-attachments/assets/44573a2b-30b7-4a44-8673-4d8a86d76f07)

Reply:
It seems an admin has to enable a setting: 
https://github.com/badges/shields/issues/1659#issuecomment-384814358

Reply:
Got it, enbled, now it works.